bionty/base/_ontology.py

```diff
@@ -93,7 +93,7 @@ def filter_include_id_prefixes(terms: pronto.ontology._OntologyTerms):  # type:i
                     return list(
                         filter(
                             lambda val: any(
-                                val.id.startswith(f"{prefix}:")
```

Comment:
Why no `:`? I added `:` here because some ontology prefixes are substring of another.

I thought `prefix:` is safer?